src/test/resources/net/sf/jabref/logic/exporter/ModsExportFormatTestMultipleEntries.bib

```diff
@@ -0,0 +1,20 @@
+% Encoding: UTF-8
+
+@Article{,
+  author   = {Shruthi, Basavaradhya Sahukar and Vinodhkumar, Palani and Selvamani},
+  title    = {Proteomics: A new perspective for cancer.},
+  journal  = {Advanced biomedical research},
```

Comment:
https://sourceforge.net/p/bibutils/home/Working%20with%20MODS/

"The part tag is the only new tag defined in the <relatedItem> that isn't defined in the main mods level. For a <relatedItem type="host">, it describes where the object is in terms of volume, issue, pages, and the like. From the above example:

    <part>
       <date>1984</date>
       <detail type="volume"><number>38</number></detail>
       <extent unit="page">
           <start>879</start>
           <end>887</end>
       </extent>
    </part>

The detail is used to describe single item locators, such as volume, issue, number, section, and session."